future-apps/payroll/test/helpers/set_token_rates.js

```diff
@@ -11,8 +11,8 @@ module.exports = web3 => {
   return async function setTokenRates(feed, denominationToken, tokens, rates, when = undefined) {
     if (!when) when = await feed.getTimestampPublic()
 
-    const bases = tokens.map(token => typeof(token) === 'object' ? token.address : token)
-    const quotes = tokens.map(() => typeof(denominationToken) === 'object' ? denominationToken.address : denominationToken)
+    const quotes = tokens.map(token => typeof(token) === 'object' ? token.address : token)
+    const bases = tokens.map(() => typeof(denominationToken) === 'object' ? denominationToken.address : denominationToken)
```

Comment:
We shouldn't be inverting the base and quote here as when setting the rates in PPF the `denominationToken` (USD) will be the quote.

Reply:
Nvm, I thought this was being used in the `Payroll_rates` test